content/en/infrastructure/containers/_index.md

```diff
@@ -226,7 +229,7 @@ You can see indexed logs that you have chosen to index and persist by selecting
 
 {{< img src="infrastructure/livecontainers/errorlogs.png" alt="Preview Logs Side panel" style="width:100%;">}}
 
-## Notes and known issues
+## Additional information
```

Comment:
Neither of these seem like issues, so updating the section title to match the one on the Processes page: https://docs.datadoghq.com/infrastructure/process/?tab=linuxwindows#additional-information